ui/packages/consul-peerings/app/templates/dc/peers/index.hbs

```diff
@@ -60,8 +65,56 @@ as |sort filters items|}}
         />
       {{/if}}
 
+    </BlockSlot>
+    <BlockSlot @name="actions">
+
+
+      <ModalDialog
+        @aria={{hash
+          label="Add peer connection"
+        }}
+        @onclose={{optional this.form.reset}}
+      as |modal|>
+        <BlockSlot @name="header">
+          {{did-insert (set this 'create' modal)}}
```

Comment:
is it possible to add the create button in the scope of the modal block instead? Or is this the usual pattern of registering the component instance on the template context to be able to trigger a modal to open/close?

Reply:
Again, not totally sure if I understood correctly. The create button is being added in the scope of the modal block. But yes this is a usual pattern.